nomiclabs/hardhat
# Hardhat launcher

This is a proof of concept, but the code in `packages/hh` and the test setup can be reviewed (of course, there should be way more tests). The code in`cli/autocomplete.ts` is awful, ignore it for now.

How to test it:

- Build and globally install `packages/hh`
- `hh` won't do a lot: out of the box is pretty much an allias for `npx hardhat`
- Run `hh install-completion` and follow the instructions
- Go to some hardhat project and try the autocomplete

Things that are supported right now:

- Autocomplete of tasks and params, even if they were custom added. Params that are specific of a task should only be shown after some task is part of the command.
- Autocomplete of networks after `--network`
